catalogue/admin.py

```diff
@@ -255,6 +242,29 @@ def get_readonly_fields(self, request, obj=None):
         else:
             return ("approved",)
 
+    def changelist_view(self, request, extra_context=None):
+        list_display = (
+            "name",
+            "approved",
+            "published",
+            "allow_email",
+            "trl",
+            "description_short",
+            "get_created_by",
+            "created",
+            "lastmodified_at",
+        )
+        list_display_links = ("name",)
+        list_editable = ("published", "allow_email")
+        if is_admin_or_mod(request):
+            list_display = list_display[:1] + ("frontpage", ) +list_display[1:]
+            list_editable += ("frontpage", )
+
+        self.list_display = list_display
+        self.list_display_links = list_display_links
+        self.list_editable = list_editable
```

Comment:
why not assign directly?

Reply:
Same as in the owner swap thing. State of list_display array etc. isn't reseted after reload. So if i always append directly on self.list_display i will have 10 "frontpage" fields after 10 loads.